### PR TITLE
feat(Tooltip): adds triggerClass prop

### PR DESCRIPTION
**Pull request by aditya-kumawat**
### What does this implement/fix? Explain your changes.
- Adds triggerClass prop.
- Adds word-break in Tooltip.
...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits
- Adds triggerClass prop in Tooltip.
- Adds ellipsis support in case of extra long string.
...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: css/src/utils/utility.css [hunk 1]
```diff
@@ -12,7 +12,8 @@
 }
 
 .ellipsis {
-    white-space: nowrap;
+    /* white-space: nowrap; */
+    /* width: 100%; */
```
Comment (satyamyadav) [inline css/src/utils/utility.css]: Please remove commented code.

File: core/components/atoms/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -81,18 +74,14 @@ export class Tooltip extends React.Component<TooltipProps, IState> {
...
     } = this.props;
 
-    const classes = classNames({
-      ['Tooltip']: true
-    }, className);
-
     const tooltipWrapper = (
       <div
-        className={classes}
+        className="Tooltip ellipsis"
```
Comment (satyamyadav) [inline core/components/atoms/tooltip/Tooltip.tsx]: Why tooltip has an `ellipsis` class.

**Review by satyamyadav — CHANGES_REQUESTED** [review]
@aditya-kumawat  Please add description in commit message.

File: core/components/atoms/tooltip/Tooltip.tsx (hunk 2 above)
Comment (aditya-kumawat) [reply core/components/atoms/tooltip/Tooltip.tsx]: If the length of string is more than max-width of tooltip, that particular string will have ellipsis and other strings will be moved to next line.
Screenshot 2020-07-22 at 10 13 04 PM

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**